boards/DCM/Inc/Io/Io_IsrHalCallback.h

```diff
@@ -0,0 +1,7 @@
+#include "main.h"
```

Comment:
This header file should be removed

Reply:
```suggestion
```

Reply:
Nope I mean this entire file. The callback's prototype is already defined somewhere else so this file isn't necessary

Reply:
A ok